app/controllers/users_controller.rb

```diff
@@ -43,6 +43,6 @@ def set_user
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def user_params
-    params.require(:user).permit(:era_commons, :organization_role)
+    params.require(:user).permit(:era_commons)
```

Comment:
This might need to be changed for the profile page, but we'll deal with that later.